input/signdescriptor.go

```diff
@@ -80,6 +81,13 @@ type SignDescriptor struct {
 	// be specified.
 	SignMethod SignMethod
 
+	// POutputs specifies an optional field that can be used to pass
+	// derivation information for the PSBT to be generated. These must
+	// match exactly the outputs of the transaction that is to be signed,
+	// and should contain any derivation information required by the signer
+	// to authorize the signature in the Unknowns field.
+	POutputs []psbt.POutput
```

Comment:
Should we mention here that these outputs shouldn't contain any information otherwise expressible in the `SignDescriptor` struct? Because all of the existing fields have an existing encoding path in a `psbt.POutput`. Maybe this could be a slice of a slice of unknowns instead, to make it more clear?

Reply:
I've changed this to a slice of a slice of unknowns instead, to make it clear.

Reply:
I've updated the comments as well.